api/app.go

```diff
@@ -72,6 +88,17 @@ func GetApp(host string, port int, configPath string, debug, fast bool, logger z
 	return app, nil
 }
 
+func (app *App) getStatusCodeFromError(err error) (*status.Status, int) {
+	var statusCode int
+	st, ok := status.FromError(err)
+	if !ok {
+		statusCode = http.StatusInternalServerError
+	} else {
+		statusCode = runtime.HTTPStatusFromCode(st.Code())
+	}
+	return st, statusCode
+}
+
 // Configure instantiates the required dependencies for podium Application
 func (app *App) Configure() error {
```

Comment:
Any reason this isn't a part of GetApp? (This would justify the error return too.)

Reply:
Aligned in person. It is called inside GetApp.

Reply:
Yep, I missed that on the first pass. Make it private, then?